apps/server/src/app/ersd/ersd.controller.ts

```diff
@@ -95,31 +98,38 @@ export class eRSDController {
 
     const s3client = new S3();
 
-    const headParams = {
+    const params = {
       Bucket,
       Key,
+      Expires: 3600,
     }
 
-    const data = await s3client.getObject(headParams).promise();
-    const body = data.Body.toString('utf-8')
+    try {
+      // Generate a presigned URL using getSignedUrlPromise
+      const presignedUrl = await s3client.getSignedUrlPromise('getObject', params);
 
-    if (format === 'xml') {
-      return response.set({'Content-Type': 'text/xml'}).send(this.stripOuterBundleXML(body))
-    } else {
-      return response.set({'Content-Type': 'application/json'}).json(this.stripOuterBundleJSON(JSON.parse(body)))
+      // Redirect the client to the presigned URL
+      response.redirect(presignedUrl);
```

Comment:
With this approach, we lose the stripping of the outer bundle and we can't do that. The Polling endpoints should deliver a file with a bundle that does not include the outer bundle. So, if we're not going to strip that here, then we would need to add another file in the S3 bucket that already has the outer bundle stripped. i.e., we'd have to have two files for each version and format - one with the outer bundle and one without.

Reply:
We have transitioned to providing a dedicated file that does not have the outer bundle so that we don't have to do the stripping and can just pass through a pre-signed URL